src/core/admittance_matrix.jl

```diff
@@ -17,7 +17,7 @@ struct AdmittanceMatrix{T}
     matrix::SparseArrays.SparseMatrixCSC{T,Int}
 end
 
-Base.show(io::IO, x::AdmittanceMatrix{<:Number}) = print(io, "AdmittanceMatrix($(length(x.idx_to_bus)) buses, $(length(nonzeros(x.matrix))) entries)")
+Base.show(io::IO, x::AdmittanceMatrix{<:Number}) = print(io, "AdmittanceMatrix($(length(x.idx_to_bus)) buses, $(SparseArrays.nnz(x.matrix)) entries)")
```

Comment:
Is this change non-breaking?  Not obvious to me.

Reply:
They're equivalent. `SparseArrays.nonzeros` returns a `Vector` of the non-zeros. `SparseArrays.nnz` returns the integer count.